app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -551,8 +551,8 @@ function AcuantCapture(
     const { image, dpi, moire, glare, sharpness } = nextCapture;
     const cardType = 'cardType' in nextCapture ? nextCapture.cardType : nextCapture.cardtype;
 
-    const isAssessedAsGlare = glare < glareThreshold;
-    const isAssessedAsBlurry = sharpness < sharpnessThreshold;
+    const isAssessedAsGlare = glare < glareThreshold!;
+    const isAssessedAsBlurry = sharpness < sharpnessThreshold!;
```

Comment:
I'd be hesitant to use the `!` operator here if we're building in the expectation that the value may be `null`.

Instead, maybe we should explicitly handle the scenario that the value is in-fact `null`, e.g. `!!glareThreshold && glare < glareThreshold;`

Reply:
Yeah, I agree, using the ! operator here made me antsy.  I like the approach you suggested and will try it, thanks!